conneg-by-ap/index.html

```diff
@@ -1316,6 +1435,98 @@ <h4>list profiles</h4>
     ]
 }
         </pre>
+        <section id="qsa-altp">
+          <h4>QSA Alternate Representations Data Model Implementation</h4>
+          <p>
+            Implementations of this specification according to the QSA Functional Profiles MUST communicate their
+            alternate representations information as per the Alternate Representations Data Model. They MAY do so using
+            HTTP <code>Link</code> headers, as per the HTTP Functional Profile, or they MAY use other approaches.
+          </p>
+          <p>
+            As noted in [[RFQ8288]] (<em>Web Linking</em>)'s introduction, HTML and some other data models have
+            "well-defined concepts of linking" so QSA implementers could implement communication of Alternate
+            Representations Data Model information via HTTP body content in HTML. This would assist human <a>client</a>
```

Comment:
The way this paragraph is written, I seems to assume that QSA is about HTML ("QSA implementers could implement ... via HTTP body content in HTML"). Wouldn't that apply to HTTP conneg, too if the preferred media type is `text/html`, and what if I do QSA and ask for media type `application/pdf`? Since media types and profiles are orthogonal, we MUST be extremely careful, not to assume that one conneg implementation is geared towards one specific media type and another implementation toward others.

Reply:
Addressed in a commit that both re-writes the questionable paragraph and gives examples in JSON as well as HTML to ensure multiple HTTP body response examples are present in the doc.